compiler/passes/convert-typed-uast.cpp

```diff
@@ -2680,9 +3001,6 @@ VarSymbol* TConverter::convertVariable(const uast::Variable* node,
   // Fix up the AST based on the type, if it should be known
   setVariableType(node, varSym, rv);
 
-  // Note the variable is converted so we can wire up SymExprs later
-  noteConvertedSym(node, varSym);
```

Comment:
Why remove this ? Is it handled elsewhere now?

Reply:
Yeah I'm trying to be a bit more consistent with the naming scheme of `convert...` vs `convertAndInsert`... IDK. The visitor is inserting right now. Maybe it doesn't matter.

Reply:
IIRC, `noteConvertedSym` isn't about inserting it; it's about updating the maps from uAST things to AST things, so that if we see a mention of the variable later, we can use the dyno resolver to determine the ID it refers to, and then use the map to find the Symbol that is, and then generate a SymExpr pointing to the right thing.